### PR TITLE
[Verification 3] Feature verification conductor processing blender verification request from work queue

### PR DESCRIPTION
**Pull request by rwrzesien**
Resolves https://github.com/golemfactory/concent/issues/224.

### CODE REVIEW

File: concent_api/conductor/tests/test_integration_verification.py [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+from django.test    import override_settings
+from django.test    import TestCase
+from django.urls    import reverse
+
+from golem_messages import message
+
+from ..models       import BlenderSubtaskDefinition
+from ..models       import UploadReport
+from ..models       import VerificationRequest
+from ..tasks        import blender_verification_order
```
Comment (bartoszbetka) [inline concent_api/conductor/tests/test_integration_verification.py]: That function exist ? `ImportError: cannot import name 'blender_verification_order'` The same in `views.py`.

File: concent_api/conductor/tests/test_integration_verification.py [hunk 2]
```diff
@@ -126,17 +130,81 @@ def test_conductor_should_schedule_verification_order_task_if_all_related_upload
...
+        upload_report.save()
+
+        with mock.patch('conductor.tasks.blender_verification_request.delay') as mock_task:
+            blender_verification_request(
+                subtask_id=self.compute_task_def['subtask_id'],
+                source_package_path=self.source_package_path,
+                result_package_path=self.result_package_path,
+                output_format=BlenderSubtaskDefinition.OutputFormat.JPG.name,  # pylint: disable=no-member
+                scene_file=self.scene_file,
+            )
```
Comment (kbeker) [inline concent_api/conductor/tests/test_integration_verification.py]: pylint gives me feedback with 4 errors, please repeair them
```python
************* Module conductor.tests.test_integration_verification
E:153,26: Instance of 'str' has no 'name' member (no-member)
E:163,88: Instance of 'str' has no 'name' member (no-member)
E:177,26: Instance of 'str' has no 'name' member (no-member)
E:206,30: Instance of 'str' has no 'name' member (no-member)
```

File: concent_api/conductor/tasks.py [hunk 3]
```diff
@@ -2,13 +2,60 @@
...
+        verification_request=verification_request,
+        output_format=output_format,
+        scene_file=scene_file,
+    )
+    blender_subtask_definition.full_clean()
+    blender_subtask_definition.save()
+
+    # If there are already UploadReports corresponding to some files, the app links them with the VerificationRequest
+    # by setting the value of the foreign key in UploadReport.
+    for path in [source_package_path, result_package_path]:
```
Comment (dybi) [inline concent_api/conductor/tasks.py]: Should we check for both `source_package_path` and `source_package_path`?

File: concent_api/conductor/tests/test_integration_verification.py [hunk 4]
```diff
@@ -126,17 +130,81 @@ def test_conductor_should_schedule_verification_order_task_if_all_related_upload
...
+        upload_report.full_clean()
+        upload_report.save()
+
+        upload_report = UploadReport(
+            path=self.result_package_path,
+        )
+        upload_report.full_clean()
+        upload_report.save()
+
+        with mock.patch('conductor.tasks.blender_verification_request.delay') as mock_task:
```
Comment (dybi) [inline concent_api/conductor/tests/test_integration_verification.py]: you could use `mock_task.assert_called_once_with(*args)` to verify proper call of the mock

File: concent_api/conductor/tasks.py (hunk 3 above)
Comment (rwrzesien) [reply concent_api/conductor/tasks.py]: I don't understand the question. Do you mean `source_package_path and result_package_path` ? If yes then yes, we should check both according to issue description.

File: concent_api/conductor/tests/test_integration_verification.py (hunk 4 above)
Comment (rwrzesien) [reply concent_api/conductor/tests/test_integration_verification.py]: :+1: done

**Pull request merged**